README.md

```diff
@@ -42,6 +44,12 @@ state = YoutubeDL.download('https://www.youtube.com/watch?v=MmIWve5bUpU', format
   .call
 ```
 
+Configurable with:
+```ruby
+YoutubeDL::Command.config.executable = 'yt-dlp' # if you're using yt-dlp
+YoutubeDL::Command.config.default_options = { some_option: true }
+```
```

Comment:
I'd say put this in a new section, `### Configuration` above Events.

Reply:
Oh, and change it to `'youtube-dl'`, since yt-dlp will be the default.

Reply:
I've changed it initially to `yt-dlp`, but then changed back as it is default for your repo 😀